base/uk.ac.stfc.isis.ibex.ui.dae/src/uk/ac/stfc/isis/ibex/ui/dae/experimentsetup/timechannels/TimeChannelsPanel.java

```diff
@@ -121,4 +149,13 @@ private void clearExistingTimeRegimeViews() {
 		}
 		timeRegimeViews.clear();
 	}
+
```

Comment:
I think these lines are from a previous incarnation of the change can you delete these.